tremor-pipeline/src/op/qos/wal.rs

```diff
@@ -360,15 +410,17 @@ impl Operator for Wal {
         signal: &mut Event,
     ) -> Result<EventAndInsights> {
         let now = signal.ingest_ns;
-        // Are we currently full
+        // Are we currently full?
         let now_full = self.limit_reached()?;
         // If we just became full or we went from full to non full
         // update the CB status
         let insights = if self.full && !now_full {
+            trace!("WAL not full any more.");
             let mut e = Event::cb_restore(signal.ingest_ns);
             e.origin_uri = self.origin_uri.clone();
             vec![e]
         } else if !self.full && now_full {
+            trace!("WAL full.");
```

Comment:
I love the idea of adding an event here, would it make sense to bump this to `warn` or at least `info` it seems like a significant event.

Reply:
Set it to `warn`. Do you mean emitting a real `Event` here? I would not do that. Logging should suffice.

Reply:
oh yes just warning that it's full is fine, logging the event would probably be terrible and blow things up :D